Modules/Registration/RegistrationMethodsv4/test/itkTimeVaryingBSplineVelocityFieldImageRegistrationTest.cxx

```diff
@@ -103,6 +104,10 @@ PerformTimeVaryingBSplineVelocityFieldImageRegistration(int argc, char * argv[])
   {
     learningRate = std::stod(argv[9]);
   }
+  if (argc >= 12)
+  {
+    convergenceWindowSize = static_cast<unsigned int>(std::stoi(argv[11]));
```

Comment:
Why not directly use `std::stoul`?

Reply:
:+1: My bad; I did not know about `std::stoul`. Done in 417c01e. I will need to review the appearances in my previous PRs.